charts/opentelemetry-operator/values.yaml

```
@@ -218,7 +218,10 @@ admissionWebhooks:
   ## TLS Certificate Option 2: Use Helm to automatically generate self-signed certificate.
   ## certManager must be disabled and autoGenerateCert must be enabled.
   ## If true and certManager.enabled is false, Helm will automatically create a self-signd cert and secret for you.
-  autoGenerateCert: true
+  autoGenerateCert:
+    enabled: true
+    # true means generate cert if cert not exists only
+    ifNotExists: true
```

```suggestion
    # If set to true, new webhook key/certificate is generated on helm upgrade.
    recreate: false
```
I think recreate might be more appropriate. I see user wanting to recreate the cert when the original expires in a year (might be ok to increase the validity when we generate the cert btw).

---

Done.